pandas-stubs/core/reshape/reshape.pyi

```diff
@@ -1,16 +1,9 @@
 import numpy as np
-from pandas.core.frame import DataFrame
+from pandas import (
+    DataFrame,
+    Series,
+)
 
-def unstack(obj, level, fill_value=...): ...
-def stack(frame, level: int = ..., dropna: bool = ...): ...
+def unstack(obj: Series | DataFrame, level, fill_value: object | None = ...): ...
+def stack(frame: DataFrame, level: int = ..., dropna: bool = ...): ...
```

Comment:
I don't think that `stack` or `unstack` are public in `reshape`.  We have `DataFrame.stack()`, `DataFrame.unstack()`, etc. so I think these 2 can be deleted.

Reply:
I removed the entire module.  CHecked other function, not in docs.

Reply:
OK, good idea